lib/OpenQA/Worker/WebUIConnection.pm

```diff
@@ -276,6 +276,15 @@ sub disable {
     $self->finish_websocket_connection;
 }
 
+# define list of HTTP error codes which indicate that the web UI is overloaded or down for maintenance
+# (in these cases the re-try delay should be increased)
+my %busy_error_codes = (502 => 1, 503 => 1, 504 => 1, 598 => 1);
+
+sub _retry_delay {
+    my ($self, $is_webui_busy) = @_;
+    return $is_webui_busy ? 60 : 5;
```

Comment:
for a follow-up change please consider making that overridable by config settings